spec/ffi-vix_disk_lib/api_spec.rb

```diff
@@ -22,15 +22,8 @@
   end
 
   describe "ConnectParams" do
-    case described_class::VERSION
```

Comment:
Since we're testing with the stub library now this case didn't make sense, but if you change the Makefile to build e.g. 5.5.0 it picks up the correct ConnectParams:

```
  4) FFI::VixDiskLib::API ConnectParams has vimApiVer
     Failure/Error: expect(described_class::ConnectParams.members).to include(:vimApiVer)
     
       expected [:vmxSpec, :serverName, :thumbPrint, :privateUse, :credType, :creds, :port] to include :vimApiVer
       Diff:
       @@ -1,2 +1,2 @@
       -[:vimApiVer]
       +[:vmxSpec, :serverName, :thumbPrint, :privateUse, :credType, :creds, :port]
```

Not sure if there's a way to test multiple .so versions in the same spec

Reply:
We can structure the travis.yml to do multiple builds based on some env car, then use that env car to build the library, maybe?

Reply:
Yeah I can do that, I'll update the Makefile to pull the vddk version from an env var

Reply:
And default to 6.7.0 or something